.idea/ant.xml

```diff
@@ -5,4 +5,4 @@
     <buildFile url="file://$PROJECT_DIR$/help/fr/build.xml" />
     <buildFile url="file://$PROJECT_DIR$/build.xml" />
   </component>
-</project>
+</project>
```

Comment:
If we could retain the new line at end of the file, ie leaving unchanged I'd be happy to approve / merge, assuming the Copyright issue has been resolved @bobjacobsen ?

Reply:
Copyright is OK.